src/components/Banner/stories.tsx

```diff
@@ -19,7 +19,7 @@ export default {
   parameters: {
     layout: 'fullscreen'
   }
-} as Meta
+} as Meta<BannerProps>
```

Comment:
All of that can be removed, as I said, we already defined the type inside `Story` which is more readable.